browsers/node16.14.2-chrome103-ff101/Dockerfile

```diff
@@ -0,0 +1,77 @@
+# WARNING: this file was autogenerated by generate-browser-image.js
+# using
+#   yarn add:browser -- 16.14.2 --chrome=103.0.5060.53 --firefox=101.0.1
+#
+# build this image with command
+#   docker build -t cypress/browsers:node16.14.2-chrome103-ff101 .
+#
+#
+FROM cypress/base:16.14.2
```

Comment:
I would recommend rebuilding this image with `16.14.2-slim` as the base image, and with Edge support

`yarn add:browser -- 16.14.2-slim --chrome=103.0.5060.53 --firefox=101.0.1 --edge`